chargeOclock/Source/Models/Client.swift

```diff
@@ -47,13 +53,42 @@ class Client {
 		}
 	}
 
-	func save(completion: @escaping (Error?, Data?) -> Void) {
-		guard !isDirty else {
-			completion(oClockError.invalidData, nil)
+	func save(completion: @escaping (Error?) -> Void) {
+		guard isDirty == true else {
+			completion(oClockError.internalError)
```

Comment:
why would this be an error condition? it's not, just return as if the save succeeded, but do nothing